### PR TITLE
feat: Update wallet interface for 128-bit API

### PR DESCRIPTION
**Pull request by adamspofford-dfinity**
The default functions are changed to use the 128-bit canister functions, and the old canister functions are accessible under explicitly 64-bit functions (e.g. `.wallet_send` &rarr; `:wallet_send128`, `.wallet_send64` &rarr; `:wallet_send`).

### CODE REVIEW

**Comment by adamspofford-dfinity** [comment]
whoops, didn't mean to re-request review, github didn't show those comments

File: ic-utils/src/interfaces/wallet.rs [hunk 1]
```diff
@@ -367,12 +444,41 @@ impl<'agent> Canister<'agent, Wallet> {
...
+        destination: &'_ Canister<'agent, Wallet>,
+        amount: u128,
+    ) -> impl 'agent + AsyncCall<(Result<(), String>,)> {
+        #[derive(CandidType)]
+        struct In {
+            canister: Principal,
+            amount: u128,
+        }
+
+        self.update_("wallet_send128")
```
[the remark this answers is not in the document]
Comment (adamspofford-dfinity) [reply ic-utils/src/interfaces/wallet.rs:458]: It probably wouldn't hurt for wallet_balance, but what about get_events? Should this receive the 64-bit events and do the big match statement to convert them to 128-bit events?

[the remark this answers is not in the document]
Comment (ericswanson-dfinity) [reply ic-utils/src/interfaces/wallet.rs:458]: Yeah, let's make it so `get_events` "just works" with either the old wallet version or the new wallet version.

[the remark this answers is not in the document]
Comment (adamspofford-dfinity) [reply ic-utils/src/interfaces/wallet.rs:458]: Won't this introduce an extra roundtrip on every call, to get the version string?

**Review by ericswanson-dfinity — APPROVED** [review]

**Pull request merged**